cloud/app/server/telemetry.js

```diff
@@ -43,6 +43,99 @@ class TelemetryService {
       throw error;
     }
   }
+  async init() {
+    log.info('Initializing TelemetryService');
+    // create logs and metrics tables if they do not exist
+    try {
+      log.info('Creating logs table if it does not exist');
+      await this.clickHouseClient.command({
+        query: `
+          CREATE TABLE IF NOT EXISTS default.logs (
+            Timestamp DateTime64(9) CODEC(Delta(8), ZSTD(1)),
+            TimestampTime DateTime DEFAULT toDateTime(Timestamp),
+            TraceId String CODEC(ZSTD(1)),
+            SpanId String CODEC(ZSTD(1)),
+            TraceFlags UInt8,
+            SeverityText LowCardinality(String) CODEC(ZSTD(1)),
+            SeverityNumber UInt8,
+            ServiceName LowCardinality(String) CODEC(ZSTD(1)),
+            Body String CODEC(ZSTD(1)),
+            ResourceSchemaUrl LowCardinality(String) CODEC(ZSTD(1)),
+            ResourceAttributes Map(LowCardinality(String), String) CODEC(ZSTD(1)),
+            ScopeSchemaUrl LowCardinality(String) CODEC(ZSTD(1)),
+            ScopeName String CODEC(ZSTD(1)),
+            ScopeVersion LowCardinality(String) CODEC(ZSTD(1)),
+            ScopeAttributes Map(LowCardinality(String), String) CODEC(ZSTD(1)),
+            LogAttributes Map(LowCardinality(String), String) CODEC(ZSTD(1)),
+            INDEX idx_trace_id TraceId TYPE bloom_filter(0.001) GRANULARITY 1,
+            INDEX idx_res_attr_key mapKeys(ResourceAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+            INDEX idx_res_attr_value mapValues(ResourceAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+            INDEX idx_scope_attr_key mapKeys(ScopeAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+            INDEX idx_scope_attr_value mapValues(ScopeAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+            INDEX idx_log_attr_key mapKeys(LogAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+            INDEX idx_log_attr_value mapValues(LogAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+            INDEX idx_body Body TYPE tokenbf_v1(32768, 3, 0) GRANULARITY 8
+          )
+          ENGINE = MergeTree
+          PARTITION BY toDate(TimestampTime)
+          PRIMARY KEY (ServiceName, TimestampTime)
+          ORDER BY (ServiceName, TimestampTime, Timestamp)
+          TTL TimestampTime + toIntervalDay(3)
```

Comment:
[Their documentation](https://clickhouse.com/docs/use-cases/observability/clickstack/ingesting-data/schemas) doesn't mention this and if I understand correctly this will delete all data after three days. Why did we add this?